.travis.yml

```diff
@@ -0,0 +1,21 @@
+language: node_js
+
+node_js:
+  - "9"
```

Comment:
Is -g necessary?

Reply:
its specifying node 9 as the version. Do you mean the global flag?